Banyan/src/investigating.jl

```diff
@@ -0,0 +1,29 @@
+# I think it's actually _good_ for this file to be check in. It's good
+# to see in each commit what was being investigated.
+
+global investigating = Dict(
```

Comment:
How is this kind of analysis different from what gets turned on with our debug mode?
or is this more granular? How does this tie in with the debug mode?

Reply:
Debug mode is for high-level visibility of Banyan client-side internals. Investigating is for investigating specific kinds of issues. There should be no overlap between things that always are for debug mode (high-level visibility) and things that are only for specific investigations.